docs/configuration.rst

```diff
@@ -0,0 +1,142 @@
+.. _configuration:
+
+Configuration
+=============
+
+Bread views are configured similar to some other class-based views, by
+subclassing and setting attributes on the subclasses, then instantiating
+one and adding it to your URL config.
+
+The main class to subclass is ``Bread``::
+
+    class UserBreadView(Bread):
```

Comment:
This is nitpicky, but `MyBreadView` would be a more typical name in an example like this.